src/scheduled-query/azext_scheduled_query/custom.py

```diff
@@ -54,9 +79,23 @@ def list_scheduled_query(client, resource_group_name=None):
     return client.list_by_subscription()
 
 
-def update_scheduled_query(cmd, instance, tags=None, disabled=False, condition=None, condition_query=None,
-                           description=None, actions=None, severity=None, window_size=None,
-                           evaluation_frequency=None, mute_actions_duration=None):
+def update_scheduled_query(cmd,
+                           instance,
+                           tags=None,
+                           disabled=None,
+                           condition=None,
+                           actions=None,
```

Comment:
The parameter "actions" is already deprecated. Please replace it by other arguments.